### PR TITLE
Dat 4587 - global navigation bar tests

### PR DESCRIPTION
**Pull request by nikodamn**
* Remove test for unsupported functionality (Wikia logo is now always navigating to Fandom)
* Update selector

Reviewers:
@Wikia/west-wing

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigation.java [hunk 1]
```diff
@@ -9,6 +9,7 @@
 import com.wikia.webdriver.pageobjectsfactory.pageobject.SearchPageObject;
 
 import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigation.java]: MINOR Remove this unused import 'org.openqa.selenium.NoSuchElementException'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/testcases/globalnavigationbar/Navigating.java [hunk 2]
```diff
@@ -1,6 +1,7 @@
 package com.wikia.webdriver.testcases.globalnavigationbar;
 
 import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
 import com.wikia.webdriver.common.logging.PageObjectLogging;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/globalnavigationbar/Navigating.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.logging.PageObjectLogging'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 8 issues:
* MINOR 8 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.configuration.Configuration'.](https://github.com/Wikia/selenium-tests/blob/67fa49c6b0c914584777d6da6d2d1fcffe62ee75/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigation.java#L5) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.CommonExpectedConditions'.](https://github.com/Wikia/selenium-tests/blob/67fa49c6b0c914584777d6da6d2d1fcffe62ee75/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigation.java#L3) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'org.openqa.selenium.support.ui.WebDriverWait'.](https://github.com/Wikia/selenium-tests/blob/67fa49c6b0c914584777d6da6d2d1fcffe62ee75/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigation.java#L16) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'org.testng.annotations.DataProvider'.](https://github.com/Wikia/selenium-tests/blob/67fa49c6b0c914584777d6da6d2d1fcffe62ee75/src/test/java/com/wikia/webdriver/testcases/globalnavigationbar/Navigating.java#L12) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'org.openqa.selenium.support.ui.ExpectedConditions'.](https://github.com/Wikia/selenium-tests/blob/67fa49c6b0c914584777d6da6d2d1fcffe62ee75/src/test/java/com/wikia/webdriver/testcases/globalnavigationbar/Navigating.java#L10) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'org.openqa.selenium.support.ui.WebDriverWait'.](https://github.com/Wikia/selenium-tests/blob/67fa49c6b0c914584777d6da6d2d1fcffe62ee75/src/test/java/com/wikia/webdriver/testcases/globalnavigationbar/Navigating.java#L11) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by SebastianMarzjan** [comment]
LGTM

**Pull request merged**